worker/src/RTC/RTCP/CompoundPacket.cpp

```diff
@@ -68,6 +68,7 @@ namespace RTC
 
 			if (this->xrPacket.Begin() != this->xrPacket.End())
 			{
+				// NOLINTNEXTLINE [clang-analyzer-deadcode.DeadStores]
```

Comment:
What? Am I supposed to write that NOLINTNEXTLINE every time I write I don't know exactly what?